cogs/zalgo.py

```diff
@@ -31,17 +41,19 @@ def __init__(self, bot):
         text = self.zalgoify(text.upper(), amount)
         await self.bot.say(text)
 
-    def zalgoify(self, text, amount=3):
-        zalgo_text = ''
-        for c in text:
-            zalgo_text += c
-            if c != ' ':
-                for t, range in ZALGO_PARAMS.items():
-                    range = (round(x*amount/5) for x in range)
-                    n = min(randint(*range), len(ZALGO_CHARS[t]))
-                    zalgo_text += ''.join(sample(ZALGO_CHARS[t], n))
-        return zalgo_text
-
+        def zalgoify(self, text, amount=3):

```

Comment:
Why is this being indented?  The method call on line 41 will fail if this is like this.